lib/command/terraform/generate.rb

```diff
@@ -5,15 +5,26 @@ module Terraform
     class Generate < Base
       SUBCOMMAND_NAME = "terraform"
       NAME = "generate"
+      OPTIONS = [
+        app_option(required: true)
```

Comment:
@rafaelgomesxyz @borela 
Should this option be required? We can make it optional and create separate folders for each app? (apps can be accessed via `config.apps`)
```
├─ terraform/
|  ├─ my-app-staging/
|  │  ├─ app.tf -- GVC config in HCL
|  │  ├─ postgres.tf -- Postgres workload config in HCL
|  │  ├─ rails.tf -- Rails worload config in HCL
|  │  ├─ providers.tf -- CPLN provider config in HCL
|  │  ├─ variables.tf -- Variables config in HCL
|  │  ├─ loacals.tf   -- Local variables config in HCL
|  ├─ my-app-production/
|  │  ├─ app.tf -- GVC config in HCL
|  │  ├─ postgres.tf -- Postgres workload config in HCL
|  │  ├─ rails.tf -- Rails worload config in HCL
|  │  ├─ providers.tf -- CPLN provider config in HCL
|  │  ├─ variables.tf -- Variables config in HCL
|  │  ├─ loacals.tf   -- Local variables config in HCL
├─ controlplane.yml -- configs for overall application
```

Reply:
Discussed this with @borela and decided to make `app` option non-required:

- if it isn't provided separate folders will be created for each app defined in `controlplane.yml`.
- If it's provided only 1 folder with terraform configs for  specific application will be created